RIGS/forms.py

```diff
@@ -140,4 +140,42 @@ class Meta:
         fields = ['is_rig', 'name', 'venue', 'start_time', 'end_date', 'start_date',
                   'end_time', 'meet_at', 'access_at', 'description', 'notes', 'mic',
                   'person', 'organisation', 'dry_hire', 'checked_in_by', 'status',
-                  'collector', 'purchase_order']
+                  'collector']
+
+
+class BaseClientEventAuthorisationForm(forms.ModelForm):
+    tos = forms.BooleanField(required=True, label="Terms of hire")
```

Comment:
Incorrect, both in testing and according to the [docs](https://docs.djangoproject.com/en/1.8/ref/forms/fields/#django.forms.BooleanField), `required=True` on a BooleanField validation ensures that the `value=True` as well.